streams/src/test/java/org/apache/kafka/test/MockRocksDbConfigSetter.java

```diff
@@ -32,4 +32,9 @@ public void setConfig(final String storeName, final Options options, final Map<S
 
         configMap.putAll(configs);
     }
+
+    @Override
+    public void close(String storeName, Options options) {
+        LOG.warn("The default close was removed in 3.0.0 -- you should overwrite it if you have implemented RocksDBConfigSetter");
```

Comment:
I think this mock is probably only used internally by Streams tests, so we don't need to log anything for users here